sphinx_gherkindoc/cli.py

```diff
@@ -35,6 +36,16 @@ def process_args(
 
     non_empty_dirs: Set[pathlib.Path] = set()
 
+    # Set parsers once and pass along where they are needed.
+    url_parser = None  # noqa: E731
+    dir_display_name_parser = None
+    for entry_point in pkg_resources.iter_entry_points("parsers"):
+        if entry_point.name == "url":
+            url_parser = entry_point.load()
+
+        if entry_point.name == "dir_display_name":
+            dir_display_name_parser = entry_point.load()
```

Comment:
NB: I did have a notion that this code should say something if an entry point was not recognized (mispelt perhaps?),
but that could also prevent other kinds of extension/expansion, so I think this is good as is.

Reply:
Yeah I'm not sure we can make assumptions about what other parsers should or shouldn't be there, so I think it is safe to let it just not work if there is a typo